docs/gettingStarted/Arena-Environment-Guide.md

```diff
@@ -0,0 +1,310 @@
+- `R` - reset the arena (cycles to the next episode if `canResetEpisode` is `true`)
+- `Q` - quit (exits the application upon press)
+
+<table>
+  <tr>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-hedgehog.png" width="500"/>
+    <p>Hedgehog</p></td>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-panda.png" width="500"/><p>Panda</p></td>
+    <td><img src="../../docs/figs/prefabs/Agent-Skins/agent-pig.png" width="500"/><p>Pig</p></td>
+</table>
+
+### Agent HUD (Heads-Up Display)
+
+The agent has a HUD (_Heads-Up Display_) that displays the following information per episode:
+
+- **Health**: The health of the agent, which is a value between `0` and `1`. The agent's health decays over time, and is reset to `1` when the agent collects a reward. The agent's health is displayed as a blue-green-red bar at the bottom of the HUD.
+- **Reward**: The reward collected by the agent, which is a value between `-1` and `1`. The agent's reward is displayed as a text at the top of the HUD, which is updated in real-time as the agent collects rewards. It contains the previous episode's reward, as well as the current episode's reward, respectively.
+- **Episode**: The episode number, which is the number of episodes the agent has played in the arena. The episode number is displayed as a white number at the top of the HUD. **(This is a feature to be added in the future.)**
+- **Notification**: The notification displayed to the agent at the end of an episode. The notification is currently a combination of color gradients and a short animated GIF. This is an optional HUD and only appears if `showNotification` parameter is set to `true` in the configuration file. _Note that this feature has no effect on training, and is only used for playing the game._
+
+| ![](../../docs/figs/Agent-HUD/agent-health.png) | ![](../../docs/figs/Agent-HUD/agent-REWARD.png) |
+|---|---|
+| ![](../../docs/figs/Agent-HUD/notification-bad.png) | ![](../../docs/figs/Agent-HUD/notification-good.png) |
+
+### Arena/Agent Limitations
+
+The arena has a few limitations, which are as follows:
+
+1. Only a single agent _per_ arena is supported, both for play and training.
+2. The agent can only move on the ground, and cannot move on the walls.
+3. The agent cannot move through objects (except for the hot/death zones).
+4. The agent cannot jump or fly.
+5. The agent cannot pick up objects (however, this is a feature to be added in the future).
+
+### Agent Properties
+
+The agent has a Phydics component attached to it, which allows it to interact with other objects in the arena. Please read our [Background - Unity](/docs/Background-Unity.md) guide for more information. 
+
+_Essentially, you can expect that the Physics of Unity game engine are modelled to mimic our three-dimensional reality as much as possible_. The agent has the following properties:
+
+- **Scale**: The scale of the agent, which is set to `1x1x1` by default.
```

Comment:
I would exclude the default values here. We would probably forget to update them if we e.g. decided to increase angular drag to 0.1

Reply:
Yes, i thought about that too but since this information is not easily obtainable by the average user, having this presented is important. I do believe these default values are not to be changed without careful consideration and wider impact on the environment first.